cypress/utils/plugins/dashboards-assistant/commands.js

```diff
@@ -193,13 +196,34 @@ Cypress.Commands.add('stopDummyServer', () => {
   });
 });
 
-Cypress.Commands.add('sendAssistantMessage', (body) =>
-  apiRequest(`${BASE_PATH}${ASSISTANT_API.SEND_MESSAGE}`, 'POST', body)
-);
+Cypress.Commands.add('sendAssistantMessage', (body, dataSourceId) => {
+  const url = `${BASE_PATH}${ASSISTANT_API.SEND_MESSAGE}${
+    dataSourceId ? `?dataSourceId=${dataSourceId}` : ''
+  }`;
+  apiRequest(url, 'POST', body);
+});
 
-Cypress.Commands.add('deleteConversation', (conversationId) =>
-  apiRequest(
-    `${BASE_PATH}${ASSISTANT_API.CONVERSATION}/${conversationId}`,
-    'DELETE'
-  )
-);
+Cypress.Commands.add('deleteConversation', (conversationId, dataSourceId) => {
+  const url = `${BASE_PATH}${ASSISTANT_API.CONVERSATION}/${conversationId}${
```

Comment:
The same as above, prefer using `qs` argument in `apiRequest`.

Reply:
Sure, the code had been updated.